test/test_helper.rb

```diff
@@ -96,6 +96,8 @@ def self.coerce(vrp)
 
     vrp.provide_original_ids unless vrp.is_a?(Hash) # TODO: re-dump with this modification
 
+    vrp[:relations]&.each{ |r| r[:type] = r[:type]&.to_sym } # TODO: re-dump with this modification
```

Comment:
Is this meant to be in the PR ? (I mean the TODO)

Reply:
kinda, I didn't want to re-dump everything (and add another 100+ Mb) for something this small :) This coerce takes care of the conversion of old dumps but eventually we would re-dump I thought -- unless we get rid of the dumps completely before :crossed_fingers:

Reply:
I will need to redump in one of my next commits (multi_tour removes trip field within vehicles) so maybe I can dump it at this time, if this modification is on dev.

Reply:
Sure, cheers Adeline! 

By the way, if it is something we can easily "coerce" via the test_helper:coerce like here, then it is better to not to dump because of the size of the repo :(

Reply:
I know, but in my case I think we need to. Because one field was removed so for instance when we try to _duplicate_safe_ we look for a key that does not exist anymore.

We could also ignore keys when parameter is nil, but I guess it is cleaner to dump everything again.

Reply:
Yeah, if a field is removed then I am afraid we have to redump the vrp; otherwise, the methods associated with the deleted field will not exist but the fields will still be in the vrp (which eventually means bugs).

I resolve this convo, feel free to unresolve it if you want me to change something.